tests/admin_widgets/tests.py

```diff
@@ -898,6 +899,54 @@ class DateTimePickerAltTimezoneSeleniumIETests(DateTimePickerAltTimezoneSelenium
 
 
 @override_settings(PASSWORD_HASHERS=['django.contrib.auth.hashers.SHA1PasswordHasher'],
+    ROOT_URLCONF='admin_widgets.urls',
+    TIME_ZONE='America/Chicago',
+    TEMPLATES=[{
+        'BACKEND': 'django.template.backends.django.DjangoTemplates',
+        # Put this app's in DIRS to take precedence over the admin's templates dir.
+        'DIRS': [
+            os.path.join(os.path.dirname(upath(__file__)), 'false_timezone_warning_templates'),
+        ],
+        'APP_DIRS': True,
+        'OPTIONS': {
+            'context_processors': [
+                'django.template.context_processors.debug',
+                'django.template.context_processors.request',
+                'django.contrib.auth.context_processors.auth',
+                'django.contrib.messages.context_processors.messages',
+            ],
+        },
+    }]
+)
+class DateTimePickerFalseTimezoneWarningsSeleniumFirefoxTests(SeleniumDataMixin, AdminSeleniumWebDriverTestCase):
+    available_apps = ['admin_widgets'] + AdminSeleniumWebDriverTestCase.available_apps
+    webdriver_class = 'selenium.webdriver.firefox.webdriver.WebDriver'
+
+    def test_false_timezone_warnings(self):
+        """
+        Ensure that no false timezone warnings appear if the utc offset is
+        missing in the admin body tag.
+        Refs #25845.
+        """
+        self.admin_login(username='super', password='secret', login_url='/')
+
+        self.selenium.get('%s%s' % (self.live_server_url,
```

Comment:
Please use longer lines (we prefer up to 119 chars when it helps readability)
Also, I think the blank lines in the test could be omitted.